src/components/Tooltip/index.tsx

```diff
@@ -1,14 +1,29 @@
 import { transparentize } from 'polished'
-import { ReactNode, useEffect, useState } from 'react'
+import { PropsWithChildren, ReactNode, useEffect, useState } from 'react'
 import styled from 'styled-components/macro'
 import noop from 'utils/noop'
 
 import Popover, { PopoverProps } from '../Popover'
 
-export const TooltipContainer = styled.div`
-  max-width: 256px;
+export enum TooltipSize {
+  Small = '256px',
+  Large = '400px',
+}
+
+const getPaddingForSize = (size: TooltipSize) => {
+  switch (size) {
+    case TooltipSize.Small:
+      return '12px'
+    case TooltipSize.Large:
+      return '16px 20px'
+  }
+}
+
+const TooltipContainer = styled.div<{ size: TooltipSize }>`
```

Comment:
Yeah, it is, however, I have run across mixed style guides so far in our codebase when to prefix with `$`. For example, we have `Column` and `Row` components that both take `gap`. Historically, I have only prefixed with `$` to avoid setting an HTML attribute when there was a name collision.

Either way, I would be happy to change, but would be great to establish a common way forward.